components/Ink/src/MDCInkView.h

```diff
@@ -125,6 +125,31 @@ typedef NS_ENUM(NSInteger, MDCInkStyle) {
  */
 - (void)cancelAllAnimationsAnimated:(BOOL)animated;
 
+/**
+ Start the first part of spreading the ink at a particular point.
+
+ This begins by fading in the ink ripple when this method is called.
+
+ @param point The user interaction position in the view’s coordinate system.
+ @param animated to add the ink sublayer with animation or not.
+ @param completionBlock Block called after the completion of the animation.
+ */
+- (void)startTouchBeganAtPoint:(CGPoint)point withAnimation:(BOOL)animated
```

Comment:
`withAnimation` sounds like it should accept a CAAnimation object instead of a BOOL. Please consider using `animated` instead. Typical methods like this are named:

```objectivec
-startTouchBeganAtPoint:animated:completion:
```

If you want to keep the conjunctions, consider:

```objectivec
-startTouchBeganAtPoint:animated:withCompletion:
```

Reply:
See UISegmentedControl for an example.